frontend/src/components/bs/components/spaces-table/components/action-cell/action-cell.tsx

```diff
@@ -0,0 +1,20 @@
+import { IconButton } from 'components/common/common';
+import { IconName, iconNameToSrc } from 'common/enums/enums';
+
+const ActionCell = (
+  id: string,
+  onSpaceDelete: (id: string) => void,
+): JSX.Element => {
+  const handleDelete = (): void => {
+    onSpaceDelete(id);
+  };
+  return (
+    <IconButton
+      icon={iconNameToSrc[IconName.TRASH]}
```

Comment:
```suggestion
      icon={IconName.TRASH}
```
map should be in the button component